SitumWayfinding/Classes/Positioning/Internal/MarkerRenderer.swift

```diff
@@ -54,20 +77,34 @@ class MarkerRenderer {
             // disassociate markers from map (otherwise we lost reference and they get stuck in map forever)
             removeMarkerFromGoogleMaps(marker: marker)
         }
+        if isClusteringEnabled {
+            markerClustering?.removeMarkers()
+        }
         markers.removeAll()
     }
     
+    func selectGMSMarker(_ marker: GMSMarker) {
+        if isClusterGMSMarker(marker) {
+            mapView.animate(toZoom: mapView.camera.zoom + 1)
+        }
+    }
+    
```

Comment:
Este método e o de arriba non se poden unificar, non entendo moi ben a diferencia entre eles

Reply:
enton un nome mais adecuado para o método sería selectGMSClusterMarker?

Reply:
si ten máis sentido, cambiado no seguinte commit